src/applications/disability-benefits/view-payments/components/view-payments-lists/ViewPaymentsLists.jsx

```diff
@@ -60,7 +60,8 @@ class ViewPaymentsLists extends Component {
           headline="We don’t have a record of returned payments"
           status="info"
           backgroundOnly="true"
-          className="vads-u-background-color--gray-lightest"
+          className="vads-u-background-color--gray-lightest vads-u-fontsize--h3"
```

Comment:
Changes here are necessary because default header level is 3 for alert box. With the change in `ViewPaymentsHeader` from an `h2` to a `p` tag, keeping these as default would have recreated the header progression issue.